books/RayTracingTheRestOfYourLife.html

```diff
@@ -1819,7 +1819,7 @@
         bool scatter(const ray& r_in, const hit_record& rec, color& attenuation, ray& scattered)
         const override {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-            auto scatter_direction = random_in_hemisphere(rec.normal);
+            auto scatter_direction = random_on_hemisphere(rec.normal);
```

Comment:
I _think_ this is correct. If not, we need to implement `random_in_hemisphere()`.